perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -95,6 +95,13 @@ def Attach(self, vm):
                 is_unattached = not(volume.status == "in-use"
                                     and volume.attachments)
 
+        for attachment in volume.attachments:
+            if self.attach_id == attachment.get('volume_id'):
+                self.device = attachment.get('device')
```

Comment:
@kivio I think using the 'device' property from the attachments list of dict objects we get back from os-volume_attachements should be OK.

If you noticed at line 85, instead of giving a device hint, I just let OpenStack return the device path it got, then I just retrieve the actual device path once the volume is attached. I believe this is reliable, and will work for other kind of hypervisors.

For later releases of OpenStack, we will need to introduce the python-cinderclient library for volume management since these operations have been deprecated out of the nova client.